documentation/oo_deployment_guide_comprehensive.adoc

```diff
@@ -2161,9 +2161,9 @@ service cgred restart
 In order for cgroups to work correctly, you need to ensure that services are started in the correct order.
 
 === Configure Disk Quotas
-When a consumer of OpenShift Origin creates an application gear, you will need to be able to control and set the amount of disk space that the gear can consume. This configuration is located in the _/etc/openshift/resource_limits.conf_ file. The two settings of interest are the qouta_files and the quota_blocks. The usrquota setting specifies the total number of files that a gear / user is allowed to own. The quota_blocks is the actual amount of disk storage that the gear is allowed to consume — where 1 block is equal to 1024 bytes.
```

Comment:
The line is too long for GitHub to highlight the change.  The change was "qouta_files" to "quota_files".